### PR TITLE
Branch reconciliation

### PR DESCRIPTION
**Pull request by jsheppard95**
PR to reconcile differences found during 1/13/21 planned DC power outage as described in issue https://github.com/pcdshub/lcls-plc-lfe-optics/issues/15

As an immediate fix, deployed commit c337a31 to PLC but with `lcls-twincat-motion` updated to v1.4.1. This was correct, and so this PR contains the following additional library updates:

lcls-twincat-pmps: 0.9.0 -> 0.13.1
ads-ioc: 0.2.5 -> 0.3.1

Project built locally and on plcprog-console without error. Still need to rebuild/deploy IOC after updating PLC configuration with this version.

Let me know if this sounds okay and I can make the PLC/IOC changes, or if there are additional changes we should have.

### CODE REVIEW

File: iocBoot/ioc-lfe-optics/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.2.5
+IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.3.1
```
Comment (klauer) [inline iocBoot/ioc-lfe-optics/Makefile:1]: While you're at it, would you mind bumping to R0.4.0?

Comment (jsheppard95) [reply iocBoot/ioc-lfe-optics/Makefile]: Can do, resolved in commit 54f078e

Comment (klauer) [reply iocBoot/ioc-lfe-optics/Makefile]: Great! You'll have to rebuild, though, and you will see some database changes (or rather fixes to make ads-ioc play more nicely with the archiver).

Comment (jsheppard95) [reply iocBoot/ioc-lfe-optics/Makefile]: Makes sense, rebuilding pointed out some duplicate log records `$(PREFIX)LCLSGeneral:GlobalLogTrickleTrip_RBV (2)`, but updating to lcls-twincat-general to `v2.4.2` resolved this. Then had to shorten arbiter PVs to follow convention and remain within 60 character limit. IOC now building without error.

**Review by klauer — APPROVED** [review]
The changes and generated records now look good to me. 👍

**Review by ZLLentz — APPROVED** [review]
LGTM!

**Pull request merged**